src/main/java/im/xiaoyao/presto/ethereum/EthereumERC20Token.java

```diff
@@ -0,0 +1,60 @@
+package im.xiaoyao.presto.ethereum;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum EthereumERC20Token {
```

Comment:
hopefully theres a better way to do this,
 with the current rate of new token and other erc generation this will get outdated pretty fast.
But nice list!

Reply:
@kaustavha thanks! I totally agree. I was lazy when I implement this and didn't source these info from an exchange for example.